runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -190,6 +191,7 @@ object GrpcClientSettings {
       None
 }
 
+@ApiMayChange
```

Comment:
To mark the settings is a bit broad, API-wise there is mostly always a way forward. But settings might become irrelevant, of course.

Reply:
I mostly meant to mark this constructor ApiMayChange, since not doing that would mean adding additional bincompat constructors each time a setting is added.